lib/moirai/engine.rb

```diff
@@ -10,10 +10,16 @@ class Engine < ::Rails::Engine
 
     config.after_initialize do
       I18n.original_backend = I18n.backend
-      if ActiveRecord::Base.connection.data_source_exists?("moirai_translations") || ENV["RAILS_ENV"] == "test"
-        I18n.backend = I18n::Backend::Chain.new(I18n::Backend::Moirai.new, I18n.backend)
-      else
-        Rails.logger.warn("moirai disabled: tables have not been generated yet.")
+
+      begin
+        if ActiveRecord::Base.connection.data_source_exists?("moirai_translations") || ENV["RAILS_ENV"] == "test"
+          I18n.backend = I18n::Backend::Chain.new(I18n::Backend::Moirai.new, I18n.backend)
+          Moirai::TranslationSync.new.synchronize
+        else
+          Rails.logger.warn("moirai disabled: tables have not been generated yet.")
+        end
+      rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid
+        Rails.logger.warn("moirai disabled: database not ready.")
```

Comment:
I've had to add this because we were getting errors on the CI where it was trying to call `synchronize`... I tried to fix by migrating etc but it wasn't working :(